app/redux/ui/actions.ts

```diff
@@ -14,7 +15,10 @@ export const setOsTheme = () => async (dispatch: AppThDispatch) => {
   dispatch({ type: SET_OS_THEME, payload: { isDarkTheme } });
 };
 
-export const switchTheme = () => ({ type: THEME_SWITCHER });
+export const switchTheme = (type: number) => ({
```

Comment:
Seems we need to store the last value of it in a persistent store to use this value on the next app session.

Only renderer requires this piece of data, so I think you can just use `localStorage` for now as the easiest solution.
If we will need this data in the main process we can easily switch to storing it on fs (via IPC and `StoreService`).

Mb write a tiny middleware for it :)

Reply:
yes, it just was a working solution for testing, replacing this solution with local storage.